dev/com.ibm.ws.jca.cm/src/com/ibm/ws/jca/cm/AbstractConnectionFactoryService.java

```diff
@@ -118,6 +121,12 @@
      */
     private ServiceReference<?> recoveryAuthDataRef;
 
+    /**
+     * MQ Queue manager id function is enabled by default. This will automatically be disabled if
+     * this function is not required by the resource adapter.
+     */
+    private boolean qmidenabled = true;
```

Comment:
Remove this from AbstractConnectionFactoryService. It will break the load-from-application path for JDBC.
All of your logic should be in ConnectionFactoryService such that it is for JCA only.

Reply:
Moved it